src/main/groovy/edu/oregonstate/mist/locations/frontend/LocationsFrontEndApplication.groovy

```diff
@@ -37,15 +35,15 @@ class LocationsFrontEndApplication extends Application<LocationsFrontendConfigur
      */
     @Override
     public void run(LocationsFrontendConfiguration configuration, Environment environment) {
```

Comment:
It looks like you updated the skeleton.
You should add these lines from the skeleton here:
```
BuildInfoManager buildInfoManager = new BuildInfoManager()
environment.lifecycle().manage(buildInfoManager)
environment.jersey().register(new NotFoundExceptionMapper())
environment.jersey().register(new GenericExceptionMapper())
environment.jersey().register(new LoggingExceptionMapper<WebApplicationException>(){})
environment.jersey().register(new PrettyPrintResponseFilter())
environment.jersey().register(new InfoResource(buildInfoManager.getInfo()))
```